docs/dev/checks/builtins/containers.md

```diff
@@ -0,0 +1,38 @@
+# containers
```

Comment:
```suggestion
# Containers
```

Reply:
We're using the name of the python module as the title (see other modules)